wiki/osu!_wiki_Contribution_Guide/Common_Issues/en.md

```diff
@@ -48,6 +48,25 @@ Fortunately, there are a few ways to resolve this problem depending on which bra
 
 `master` is the main branch of your fork repository. It should contain a clean copy of `osu-wiki` repository's `master` branch contents without any personal edits. There are two ways to update the `master` branch of your fork:
 
+#### Using GitHub
```

Comment:
while I appreciate you going into details (not kidding), it'd be easier if this was a go-to set of instructions as conscise as possible. besides, if we have this, we don't need upriver at all.
* * *
also, don't forget to outdate the existing translations!

Reply:
I managed to make it concise as possible and make it idiot-proof, however, you may try your hand on making it more understandable (it's pretty hard to make it understandable enough to most people (I applaud those people who make user guides for this purpose)).